google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -1795,25 +1830,32 @@ public void testWriteLogEntriesWithSeverityFlushEnabled() {
 
   @Test
   public void testWriteLogEntriesWithOptions() {
-    Map<String, String> labels = ImmutableMap.of("key", "value");
-    WriteLogEntriesRequest request =
-        WriteLogEntriesRequest.newBuilder()
-            .putAllLabels(labels)
-            .setLogName(LOG_NAME_PB)
-            .setResource(MONITORED_RESOURCE.toPb())
-            .addAllEntries(
-                Iterables.transform(
-                    ImmutableList.of(LOG_ENTRY1, LOG_ENTRY2), LogEntry.toPbFunction(PROJECT)))
-            .build();
-    WriteLogEntriesResponse response = WriteLogEntriesResponse.newBuilder().build();
-    EasyMock.expect(loggingRpcMock.write(request)).andReturn(ApiFutures.immediateFuture(response));
-    EasyMock.replay(rpcFactoryMock, loggingRpcMock);
-    logging = options.getService();
-    logging.write(
-        ImmutableList.of(LOG_ENTRY1, LOG_ENTRY2),
-        WriteOption.logName(LOG_NAME),
-        WriteOption.resource(MONITORED_RESOURCE),
-        WriteOption.labels(labels));
+    test_write_log_entries_with_destination(
```

Comment:
Please fix the method name `test_write_log_entries_with_destination` to follow [Java style](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names) for method names.